### ISSUE TITLE
How to save the name of a feature when saving model feature importance

### ISSUE THREAD

**Issue by hzy0110** [microsoft/qlib#1836]
hello 
I use the following code

`with R.start(experiment_name=EXP_NAME):
    mlflow.lightgbm.autolog()
    R.log_params(**flatten_dict(model._params)) # 用于记录参数
    model.fit(dataset=ds) 
`
Feature importance is saved
But the saved name is  Column_X
What I want saved is the factor name, such as ROC5

I know it can be obtained via 
`ds.handler.get_feature_config()`
But get_feature_config can only get feature_importance_split value, but not the value of feature_importance_gain value
How to save feature importance without using Columns_XX names

**Comment by glidis** [comment]
A simple workaround is as follows:

1. Locate the line:
    `ds_l.append((lgb.Dataset(x.values, label=y, weight=w), key))`
    in `qlib/contrib/model/gbdt.py`.
2. Remove .values from x.values.
3. Save the file and reload.

The issue arises because x.values strips the column names from the DataFrame x, which causes the problem.
